docs/building_fledge/04_utilities.rst

```diff
@@ -14,7 +14,7 @@
 
 .. |GH Issues| raw:: html
 
-   <a href="https://github.com/fledge/Fledge/issues" target="_blank">GitHub</a>
+   <a href="https://github.com/fledge-iot/Fledge/issues" target="_blank">GitHub</a>
```

Comment:
fyi, Fledge issues option is NOT available on GitHub for any user. May be they have disabled or turn issues off for the repository